design/ceph/ceph-external-cluster.md

```diff
@@ -157,3 +157,43 @@ monitoring:
 4. Create a new metric Service
 
 5. Create an Endpoint resource based out of the IP addresses either discovered or provided by the user in the script
+
+## Steps for creating external rook-ceph cluster
+
+1) Create a standalone [ceph cluster](https://docs.ceph.com/en/pacific/cephadm/#cephadm).
```

Comment:
I don't think this step is needed. We just assume there is a cluster we want Rook to connect to.